### DISCUSSION TITLE
how to interpret I2CDriver

### DISCUSSION THREAD

**Discussion by jeffbrown57** [arendst/Tasmota#21316] [Q&A]
when i run i2cdriver on my esp32 ( built with tasmota-sensors.ban , it returns 
21:36:16.926 RSL: RESULT = {"I2CDriver":"7,8,9,10,11,!12,13,14,15,17,18,20,24,29,31,36,41,42,44,46,48,58,62,65,69,76,77,82"}
i'm trying to get a AHT21 temp,hum sensor which is driver 43 ? so if i don't see ,43, above thats a problem , or am i interpreting this wrong ?

**Comment by sfromis** [comment]
It means that you are not using a custom [build](https://tasmota.github.io/docs/BUILDS/) with `USE_AHT2x` to include this sensor. And dont use `tasmota-sensors`; the sensors included there on ESP8266 are **already** included in ESP32 builds. But lots of supported sensors are only included with a custom build.

**Reply by jeffbrown57** [reply]
ok thanks

**Reply by jeffbrown57** [reply]
for purposes of custom compiling using GitPod , does tasmota concern itself with the difference between esp8266 and esp32 , my target to flash will be an esp32 , but the custom firmware i built has esp8266 all over it , is there somewhere in the source or platformio i can specify esp32 ?  sorry if this is an ignorant question  ....

**Reply by jeffbrown57** [reply]
sorry never mind, i figured it out ,, platformio run -e tasmota32

**Reply by HofeBY** [reply]
The driver for AHT21 was improved just three days ago.
It is therefore important to use the very latest development source branch.
Most compiled versions probably do not contain this change.
